code/Blender/BlenderLoader.cpp

```diff
@@ -420,7 +420,7 @@ void BlenderImporter::ConvertBlendFile(aiScene* out, const Scene& in,const FileD
 // ------------------------------------------------------------------------------------------------
 void BlenderImporter::ResolveImage(aiMaterial* out, const Material* mat, const MTex* tex, const Image* img, ConversionData& conv_data)
 {
-    (void)mat; (void)tex; (void)conv_data;
+    (void)mat;
```

Comment:
You can jist remove it out of the parameter list: void BlenderImporter::ResolveImage(aiMaterial* , ...